src/renderer/services/css.js

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
I think we can use the `classnames` npm module, it does exactly the same thing :'(

Reply:
Resolved